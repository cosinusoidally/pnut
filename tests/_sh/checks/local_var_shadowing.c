// expect_comp_failure
void main() {
  int a;
  int a;
}
