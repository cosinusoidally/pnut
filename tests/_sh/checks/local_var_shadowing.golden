tests/_sh/checks/local_var_shadowing.c:6:0  Variable is already in local environment
